test/release/examples/primers/interopWithC.suppressif

```diff
@@ -0,0 +1,4 @@
+# This test currently gives this:
+# /usr/local/cuda-11.8/include/thrust/functional.h:598: error: 'square' has multiple definitions, redefined at:
+#   /usr/local/cuda-11.8/include/thrust/functional.h:598
+CHPL_GPU == nvidia
```

Comment:
This is based on a paratest on chapdl-nvidia?

I wonder if the same issue is still there with newer CUDAs as this error message looks like a name conflict with CUDA 11's thrust. No need to pursue this for the sake of this comment, just something to consider when/if we revisit this test.

Reply:
Right. We will know when this surpressif starts passing.